screens/Authentication.jsx

```diff
@@ -0,0 +1,95 @@
+import React, {useState} from 'react';
+import {
+  Button,
+  StyleSheet,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+
+const Authentication = () => {
+  const [phone_number, setPhoneNumber] = useState('');
+  const handlePhoneNumberChange = number => {
+    const formatted_number = number.replace(/[^0-9]/g, '');
+    if (number.length <= 10) {
+      setPhoneNumber(formatted_number);
+    }
+  };
+  return (
+    <View style={styles.container}>
+      <View>
+        <Text style={styles.title}>Verify your mobile number </Text>
+        <Text style={styles.subtitle}>
+          Enter your number so that we know you're you
+        </Text>
+      </View>
+
+      <View style={styles.inputContainer}>
+        <TextInput
+          style={styles.phone_input}
+          placeholder="Enter your mobile number"
+          value={phone_number}
+          onChangeText={handlePhoneNumberChange}
+          keyboardType="phone-pad"
+        />
+      </View>
+      <View tsyle={styles.buttonContainer}>
+        <TouchableOpacity
+          style={styles.button}
+          onPress={() => {
+            console.log(phone_number);
+          }}>
+          <Text style={styles.buttonText}>Continue</Text>
+        </TouchableOpacity>
+      </View>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    padding: 30,
+    height: '100%',
```

Comment:
try to  calculate the height with the dimension library from the react-native rather than using percent